multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -12,7 +12,8 @@
     BINARY,
     CATEGORICAL,
     CLASSIFICATION,
-    IMAGE,
+    IMAGE_PATH,
```

Comment:
`IMAGE` is used in the `data_types` in the model configs to indicate one model requires image data. `IMAGE_PATH` is to be specific that the dataframe contains image paths. There may be other kinds of formats for image data, e.g., images encoded in base64 or image indexes. By default, we assume image paths if not specified otherwise.

Reply:
Currently do we support other type of image input expect for image_path?